code/game/objects/items/melee/f13twohanded.dm

```diff
@@ -33,13 +33,13 @@
 // AXES //
 //////////			-bonus damage to all doors, windows
 
-// Legion Axe		Keywords: Damage 47/60
+// Legion Axe		Keywords: Damage 30/60 do not change these numbers
 /obj/item/twohanded/legionaxe
 	name = "War Honed Axe"
 	desc = "Heavy fireman axe from the old world, Restored to working order by legion craftsmen. Excellent for smashing doors or heads."
 	icon_state = "legionaxe"
 	icon_prefix = "legionaxe"
-	force = 47
+	force = 30
```

Comment:
You forgot to change line 54. Set "force_unwielded" to 30 as well.

Reply:
Thanks fixed